src/typescript-node/manifest.json

```
@@ -0,0 +1,60 @@
+{
+	"variants": ["18-bullseye", "16-bullseye", "14-bullseye", "18-buster", "16-buster", "14-buster"],
+	"definitionVersion": "0.204.2",
```

```suggestion
	"imageVersion": "0.204.2",
```

`imageVersion` is what is now used by the vscdc tooling.